immuneML/encodings/distance_encoding/EditDistanceEncoder.py

```diff
@@ -0,0 +1,298 @@
+from pathlib import Path
+import subprocess
+import os
+import pandas as pd
+import numpy as np
+import warnings
+from tempfile import NamedTemporaryFile
+
+
+from immuneML.data_model.dataset.RepertoireDataset import RepertoireDataset
+from immuneML.data_model.encoded_data.EncodedData import EncodedData
+from immuneML.encodings.DatasetEncoder import DatasetEncoder
+from immuneML.encodings.EncoderParams import EncoderParams
+from immuneML.environment.EnvironmentSettings import EnvironmentSettings
+from immuneML.util.EncoderHelper import EncoderHelper
+from immuneML.util.ParameterValidator import ParameterValidator
+
+
+class EditDistanceEncoder(DatasetEncoder):
```

Comment:
is there a specific reason why it's called EditDistance, but computes Morisita-Horn? :) is that configurable or should it always be MH?